packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -181,6 +183,22 @@ class MenuContent extends React.Component {
     }
   }
 
+  getSubmenuHeight() {
+    if (this.props.index > 0 && this.listNode) {
+      const bufHeight = (this.context.name || this.context.className) === 'orion-fusion-theme' ? 20 : 10;
+      const submenuHeight = this.listNode.clientHeight + this.listNode.parentNode.parentNode.parentNode.firstChild.clientHeight + bufHeight;
```

Comment:
Curious, listNode.clientHeight + childHeight (if header is available) should be enough to find the height of the menu !! why do we need childHeight (for when header was not present )+ buffHeight.

Reply:
`bufHeight` would be necessary here to add for the divider space between header(firstChild.clientHeight) and the first menu item. Header will always be there since we’re setting height only for submenus.